rclcpp/test/rclcpp/test_qos_parameters.cpp

```diff
@@ -234,7 +234,8 @@ TEST(TestQosParameters, declare_no_parameters_interface) {
 
   std::map<std::string, rclcpp::Parameter> qos_params;
   EXPECT_FALSE(
-    node->get_node_parameters_interface()->get_parameters_by_prefix("qos_overrides", qos_params));
+    node->get_node_parameters_interface()->get_parameters_by_prefix(
+      "qos_overrides./my/fully/qualified/topic_name", qos_params));
```

Comment:
I don't understand why this changed either (probably just missing something), but this might be an indicator that we're breaking behavior and therefore we need documentation of this, in the migration guide for galactic and/or user docs.

Reply:
Before, the only parameters that started with the prefix `qos_overrides` were the ones immediately declared above.
Now there's also the ones of the `/parameter_events` topic